src/effects/backends/audiounit/audiounitmanager.mm

```diff
@@ -17,6 +19,8 @@
         return;
     }
 
+    dispatch_group_enter(m_instantiationGroup);
+
```

Comment:
the doc says `dispatch_group_enter` must balanced with a `dispatch_group_leave` call. This is only the case in the `instantiateAudioUnitSync` case, not in the `instantiateAudioUnitAsync` case. Is this a bug or is there a reason for that?

Reply:
In both cases the `instantiateWith` method is called, which includes the leave call (see below). In the async case `initializeWith` is called from the completion callback.